src/Fable.Transforms/Fable2Babel.fs

```diff
@@ -3797,6 +3798,12 @@ module Util =
                 |> List.mapi (fun index arg ->
                     let name = defaultArg arg.Name $"arg{index}"
 
+                    /// Try to find getter/setter in f# syntax for POJOs. If found propagate its xml doc to interface.
+                    let tryXmlDoc =
+                        ent.MembersFunctionsAndValues
```

Comment:
This lookup may be a bit brittle when there are overloads. I know we don't normally allow overloads on interfaces, unless the mangle attribute is added, so it should be mostly fine, at least as a first pass.